Capri.Web/Controllers/AccountController.cs

```diff
@@ -19,9 +19,9 @@ public AccountController(ILoginService loginService)
         [HttpPost]
         public async Task<IActionResult> Login([FromBody]UserCredentials credentials)
         {
-            if (credentials == null)
+            if(credentials == null || !ModelState.IsValid)
             {
-                return BadRequest("Credentials not given");
+                return BadRequest("You provided invalid data");
```

Comment:
I think using impersonal construction like `Invalid credentials provided` will be better, because previosly you've been using it mostly.

Reply:
Done